scio-core/src/main/scala/com/spotify/scio/hash/ApproxFilter.scala

```diff
@@ -355,3 +347,41 @@ object BloomFilter extends ApproxFilterCompanion {
     new BloomFilter[T](impl)
   }
 }
+
+class ScalableBloomFilter[T: g.Funnel] private (private val impl: MutableScalableBloomFilter[T])
+    extends ApproxFilter[T] {
+
+  /** Return an estimate for the total number of distinct elements that have been added to this [[ApproxFilter]]. */
+  override def approxElementCount: Long = impl.approximateElementCount
+  override def mightContain(elem: T): Boolean = impl.mightContain(elem)
+  override val expectedFpp: Double = impl.fpProb
+}
+
+// TODO this has questionable value; the API has already materialized `elems`, so you can with less overhead just create a single normal BF
```

Comment:
That's true, maybe it's indeed overkill to fit `ApproxFilter` here, plus the mutable nature doesn't really fit `AF` 's immutable design. Should we undo that?

Reply:
Yeah will do.